### ISSUE TITLE
Duplicate MyHealtheVet coordinator accordians

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#5456]
This issue was originally titled **[AXE-CORE]: IDs on the page must be unique** but this is more than an accessibility issue. 

## Description

We have two of the same accordion on the page for MyHealtheVet Coordinator `
Related to accessibility, it means we also have two <div>` content blocks, causing an axe error. This can cause assistive tech -- JAWS particularly -- to mis-behave with aria-controls attributes. Screenshot attached below.

## Type of Issue
Content

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria
[ ] As a Veteran, I want all accordions to be unique so that I am not confused by the webpage content. 

* [ ] The axe error should not appear on future scans

## Environment

* Chrome latest
* https://www.va.gov/pittsburgh-health-care/locations/beaver-county-va-clinic/

## Possible Fixes (optional)

Removing the duplicated accordion or updating the ID should remove the issue.

## WCAG or Vendor Guidance (optional)

* [F77: Failure of Success Criterion 4.1.1 due to duplicate values of type ID](https://www.w3.org/TR/WCAG20-TECHS/F77.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-03 at 3.44.45 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/259fc305-a2c3-4334-b9a9-c326b81c276c)

**Comment by davidconlon** [comment]
Caused by duplicative nodes 1015 and 2335.  Pinged CMS team via slack to determine who should delete duplicative nodes (Facilities, CMS, VAMC, VHA-Owners) going forward.

**Comment by davidconlon** [comment]
I deleted node 2335 per the instructions of CMS team.

**Comment by jenstrickland** [comment]
Validated that the duplicate content is now removed.

![Screen Shot 2020-03-11 at 2 21 33 PM](https://user-images.githubusercontent.com/57469/76450103-a3f84f80-63a3-11ea-8fca-1ae0820a14d7.png)
